gradio/components.py

```diff
@@ -2356,6 +2356,8 @@ class Dataframe(Changeable, IOComponent):
     Demos: filter_records, matrix_transpose, tax_calculator
     """
 
+    md = None
```

Comment:
IMO, It'd be better for understandability if we rename this variable to `markdown` or `has_markdown`. WDYT @pngwn ?

Reply:
Good point, this is actually the markdown parser instance. I'll rename it accordingly.